tools/invocation_header_gen.py

```diff
@@ -14,8 +14,8 @@
 import sys
 import xml.dom.minidom
 import pkg_resources
-# We require jinja2 to be at least version 2.10 as we use the 'namespace' feature from
-# that version
+# We require jinja2 to be at least version 2.10,
+# so we can use use the 'namespace' feature from that version.
```

Comment:
That comment feels a bit wrong if namespaces are no longer used once the changes are merged. Maybe put the commit comment in here, it it's clear that being able to use `minijinja` is the reason certain features are not used.

Reply:
Yeah, its a bit of a weird state, other templates still use `namespace`,  In theory in the future I wanted to keep open that sel4 could add more namespace features in the future, even though `minijinja` doesn't support it there is no proposal to actually use minijinja in tree.

In theory this version of the template would allow us to downgrade the version requirements, I didn't want to do that in case there was a need for this feature in the future.

Reply:
I'll try and reword this so that intent is better reflected.